pkg/pfpstatus/record/recorder.go

```diff
@@ -223,7 +257,7 @@ func (rr *Recorder) Push(st podfingerprint.Status) error {
 	}
 
 	if !ok {
-		nr, err = NewNodeRecorder(st.NodeName, rr.timestamper, WithCapacity(rr.nodeCapacity))
+		nr, err = NewNodeRecorder(st.NodeName, WithTimestamper(rr.timestamper), WithCapacity(rr.nodeCapacity))
```

Comment:
I like the new uniform API, I'm just thinking about when the options grow more (like https://github.com/openshift-kni/debug-tools/pull/98)  we always need to adjust this line and NewNodeRecorder function. When we call this function from the Recorder.Push we know for sure that the new node recorders will "inherit" the same options as in Recorder. Should/can we make it simpler by introducing new recorder field "Options" and setting up new node recorder by passing this "rr.Options"? that in turn will move the interpretation of recorderOptions to NodeRecorderOptions to an inner function. Is that something we want to do or keep this version for better readability?